tests/cupyx_tests/scipy_tests/ndimage_tests/test_filters.py

```diff
@@ -407,15 +416,15 @@ def test_filter(self, xp, scp):
             'filter': ['maximum_filter', 'minimum_filter'],
             'origin': [0, (0, 1, 1, 0)],
             'kshape': [(3, 5, 3, 5)],
-            'mode': ['reflect'],
+            'mode': ['constant'],
```

Comment:
Ditto (regarding the filter support for `reflect`)

Reply:
several modes are supported and tested by the case above. This specific case is mainly intended to test specific kernel shapes (`kshape`) so I don't think we need to repeat that for every supported mode. I used `'constant'` instead of 'reflect' to also test the case with per-axis `cval` argument.